### PR TITLE
Shopper may add Notes to a Product in their Cart on checkout

### LINKED ISSUE #2661: Marketplace: Shopper may add Notes to Products in their Cart on Checkout
So, I've done some pondering on how I want this to work, but I'm thinking that on the Checkout screen, where we show the Shopper the items in their Cart and collect their Delivery Address and use Turbo to swap the element with a small form that lets them adjust quantity, remove, or add notes for the particular item.

This way folks can say "I want the Pastrami" when they're ordering a sandwhich from Oaklandia.

### PR DESCRIPTION
**Pull request by zspencer**
- Fixes #2661

#### Email Receipt:
Screenshot 2024-10-16 at 6 19 54 PM

#### Checkout Page
Screenshot 2024-10-16 at 6 20 19 PM

##### Editing Note

Screenshot 2024-10-16 at 6 20 23 PM

### CODE REVIEW

File: app/furniture/marketplace/cart_product/notes/edit.html.erb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
```
Comment (anaulin) [inline app/furniture/marketplace/cart_product/notes/edit.html.erb:1]: unnecessary blank line

File: app/furniture/marketplace/cart_product/notes/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,4 @@
+
```
Comment (anaulin) [inline app/furniture/marketplace/cart_product/notes/new.html.erb:1]: unnecessary blank line

**Review by anaulin — APPROVED** [review]
Thank you!

File: app/furniture/marketplace/cart_product/notes/new.html.erb (hunk 2 above)
Comment (zspencer) [reply app/furniture/marketplace/cart_product/notes/new.html.erb]: These are the most necessary blank lines!

File: app/furniture/marketplace/cart_product/notes/edit.html.erb (hunk 1 above)
Comment (zspencer) [reply app/furniture/marketplace/cart_product/notes/edit.html.erb]: Removed 😅

**Pull request merged**